### DISCUSSION TITLE
LSP4IJ Discussion

### DISCUSSION THREAD

**Discussion by Eveeifyeve** [SkriptLang/Skript#7208] [General]
Discussion moved from #6447.

**Comment by Eveeifyeve** [comment]
I could see LSP4IJ having some support @angelozerr, because of both the intellij CE and Ultimate Support but I could see adding other ides like rustrover and etc could be a problem.

**Reply by angelozerr** [reply]
Lsp4ij should work with rustrover, clion, goland etc

**Comment by Eveeifyeve** [comment]
But I will add support of IntelliJ and then add anonymous telemetry to see how many people are using Intelij compared to something like vscode or zed.
